drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -85,7 +85,12 @@ uint16_t at86rf2xx_get_addr_short(at86rf2xx_t *dev)
 void at86rf2xx_set_addr_short(at86rf2xx_t *dev, uint16_t addr)
 {
     dev->addr_short[0] = addr >> 8;
-    dev->addr_short[1] = addr & 0xff;
+    dev->addr_short[1] = addr;
+#ifdef MODULE_SIXLOWPAN
+    /* https://tools.ietf.org/html/rfc4944#section-12 requires the first bit to
+     * 0 for unicast addresses */
+    dev->addr_short[1] &= 0x7F;
```

Comment:
I am just wondering: how do we know that `addr` is not a multi-cast address here? This feature doesn't seem to be implemented though, and the RFC is also somewhat less informative regarding multi-cast addresses